packages/client/styles/helpers/cardRootStyles.ts

```diff
@@ -7,7 +7,6 @@ const cardRootStyles = {
   borderRadius: Card.BORDER_RADIUS,
   boxShadow: cardShadow,
   minWidth: 256,
-  maxWidth: 296,
```

Comment:
update: gonna revert this change because the # of people affected by the extra long boys is probably greater than the number of people affected by the Task drawer